core/utils/nodes.ts

```diff
@@ -0,0 +1,10 @@
+import IsNumberNode from '../rules/IsNumberNode';
+import { convertToBool } from './convert';
+
+export function getNodeByName(name: string, value: unknown) {
+  if (name === 'isNumber') {
```

Comment:
Needs to be changed to Map or another data structure in future, because there will be a lot of different options supported by our schemas.

Reply:
Will be fixed in next issues